package.json

```diff
@@ -26,6 +26,7 @@
     "dependencies": {
         "@headlessui/react": "1.7.17",
         "@heroicons/react": "2.0.18",
+        "@material-tailwind/react": "2.1.10",
```

Comment:
I would suggest to use https://www.radix-ui.com/ instead so we don't use multiple component libraries (I have already used the slider primitive one from them).
I was thinking about switching over to them for every UI component so we don't have to look for custom solutions all the time. What do you think?

Reply:
You were right, I didn't use this module at the end 😶‍🌫️ 
Haven't tried [them](https://www.radix-ui.com/) but had a look and they seem cool, let's have a ticket to switch all of them in all UI components. Can pick this up after the fav feature.